src/app/shared/entry-actions/tool-actions.component.html

```diff
@@ -28,7 +28,13 @@
     </button>
   </ng-template>
   <ng-template #enabledDeleteButton>
-    <app-deregister-modal *ngIf="!tool?.is_published" [refreshMessage]="isRefreshing$ | async"></app-deregister-modal>
+    <app-deregister-modal
```

Comment:
Should this still be here?

Reply:
Completely reverted the file to the version from develop